python/cudf/_gdf.py

```diff
@@ -141,6 +141,12 @@ def gdf_to_np_dtype(dtype):
     'hash': libgdf.GDF_HASH
 }
 
+_null_sort_behavior_api = {
+    'null_as_largest': libgdf.GDF_NULL_AS_LARGEST,
+    'null_as_smallest': libgdf.GDF_NULL_AS_SMALLEST,
+    'null_as_largest_multisort': libgdf.GDF_NULL_AS_LARGEST_FOR_MULTISORT
+}
```

Comment:
The CFFI that needs this code will be pulled out in #1533 so you can remove this.